src/Main.java

```diff
@@ -1,5 +1,129 @@
+import java.util.Arrays;
+import java.util.Scanner;
+
 public class Main {
+    static String[][] plansza = {{"-", "-", "-"},{"-", "-", "-"},{"-", "-", "-"}};
     public static void main(String[] args) {
-        System.out.println("Hello world!");
+        for(int i = 0; i<5; i++) {
+            ruch(plansza);
+        }
     }
-}
\ No newline at end of file
+
+    static void sprawdzWinO(){
+
+        if(plansza[0][0].equals("O") && plansza[0][1].equals("O") && plansza[0][2].equals("O")) {
+            System.out.println("Gracz 1 wygrał!");
+            System.exit(-1);
+        }else{
+            if(plansza[0][0].equals("O") && plansza[1][1].equals("O") && plansza[2][2].equals("O")) {
+                System.out.println("Gracz 1 wygrał!");
+                System.exit(-1);
+            }else{
+                if(plansza[0][0].equals("O") && plansza[1][0].equals("O") && plansza[2][0].equals("O")) {
+                    System.out.println("Gracz 1 wygrał!");
+                    System.exit(-1);
+                }else{
+                    if(plansza[1][0].equals("O") && plansza[1][1].equals("O") && plansza[1][2].equals("O")) {
+                        System.out.println("Gracz 1 wygrał!");
+                        System.exit(-1);
+                    }else{
+                        if(plansza[2][0].equals("O") && plansza[2][1].equals("O") && plansza[2][2].equals("O")) {
+                            System.out.println("Gracz 1 wygrał!");
+                            System.exit(-1);
+                        }else{
+                            if(plansza[0][1].equals("O") && plansza[1][1].equals("O") && plansza[2][1].equals("O")) {
+                                System.out.println("Gracz 1 wygrał!");
+                                System.exit(-1);
+                            }else{
+                                if(plansza[0][2].equals("O") && plansza[1][2].equals("O") && plansza[2][2].equals("O")) {
+                                    System.out.println("Gracz 1 wygrał!");
+                                    System.exit(-1);
+                                }else{
+                                    if(plansza[0][2].equals("O") && plansza[2][1].equals("O") && plansza[2][0].equals("O")) {
+                                        System.out.println("Gracz 1 wygrał!");
+                                        System.exit(-1);
+                                    }
+                                }
+                            }
+                        }
+                    }
+                }
+            }
+        }
+    }
+
+    static void sprawdzWinX(){
+
+        if(plansza[0][0].equals("X") && plansza[0][1].equals("O") && plansza[0][2].equals("X")) {
```

Comment:
zmień to sprawdzenie trochę,
za dużo ifów tutaj jest, zastanów się nad rozwiązaniem z pętlą która chodzi bo danym wierszu i sprawdza czy sa wpisane w ten wiersz znaki "X" lub "O"